src/ngraph/runtime/cpu/unit_test.manifest

```diff
@@ -8,6 +8,8 @@ sum_stable_acc_double
 max_3d_to_scalar_int32
 
 # Not implemented
+erf
+zero_sized_erf
```

Comment:
You didn't add these test in this PR so why are they listed here?

Reply:
Merge artifact - removed.